wallet-ext: account selector in tokens view

pchrysochoidis opened the pull request:
* under `wallet-multi-accounts` feature flag
* replaces `...` to `…` for ellipsis hook
* adds `address` to query key hash for the `useRecentTransactions` to load transactions for the current active address
* changes the way we clear objects from store and now we do it from home component until we move it to query

When only one account is available or `wallet-multi-accounts` is disabled

https://user-images.githubusercontent.com/10210143/216132976-46070a62-85e8-4555-92eb-fb7d8102e61b.mov

For multi-accounts

https://user-images.githubusercontent.com/10210143/216133460-7cff027c-9c24-4037-8aeb-be4fa3acaf53.mov


https://user-images.githubusercontent.com/10210143/216133536-c87ba39b-c8f4-43ab-8154-b4735f8184a9.mov


closes: APPS-296

mystie711 commented:
@pchrysochoidis realized one small delta in Figma that i have now updated.
For single wallet interaction, on-hover over the pill, could we also have the icon be colorized?
<img width="290" alt="image" src="https://user-images.githubusercontent.com/109102528/216135740-ebfffab6-1366-4c6f-b7ca-10e4acb7e062.png">

mystie711 reviewed (approved):
looking good.

pchrysochoidis commented:
> @pchrysochoidis realized one small delta in Figma that i have now updated. For single wallet interaction, on-hover over the pill, could we also have the icon be colorized? <img alt="image" width="290" src="https://user-images.githubusercontent.com/109102528/216135740-ebfffab6-1366-4c6f-b7ca-10e4acb7e062.png">

Updated it


https://user-images.githubusercontent.com/10210143/216184143-1f4899f3-4450-40c1-a32d-049849c96405.mov


https://user-images.githubusercontent.com/10210143/216184153-69fb0970-af86-4b94-8512-35a1f0aac4d8.mov

pchrysochoidis commented:
Thanks @mamos-mysten, I updated the animation to 200ms and added the hover state


https://user-images.githubusercontent.com/10210143/216476476-ea60a5a6-7afe-4276-9bd1-fd433a650b0e.mov

mamos-mysten reviewed (approved):
looks good to me
